internal/storage/pebble/storage.go

```diff
@@ -183,6 +185,15 @@ func (s *Storage) GetTotalSupply() *storage.TotalSupply {
 	return ptr
 }
 
+func (s *Storage) GetAccounts() []models.Account {
+	ptr := new([]models.Account)
+	err := s.getFromDB(ptr, getKey(getPrefix(ptr), "0x0", 0))
```

Comment:
I think there is no need in this "0x0", could be an empty string

Reply:
That's symbolic. Does having this key have any impact on retrieval? if not then I think we should keep it to match the getter function param headers better

Reply:
Object have prefix already, so this is just some piece of data that you don't really need there and looks like some magic string that you're passing everywhere. I think you can keep it, but I think its better to add some wrapper method to batch, so you won't be specifying this at all